components/board/intel/board.go

```diff
@@ -0,0 +1,26 @@
+// Package intel implements a Intel based board.
+package intel
+
+import (
+	"github.com/edaniels/golog"
+	"github.com/pkg/errors"
+	"periph.io/x/host/v3"
+
+	"go.viam.com/rdk/components/board/genericlinux"
+)
+
+const modelName = "intel"
```

Comment:
[major] We don't have a scop with a model name yet, so let's not register this fully just yet, as that is a breaking change.

Reply:
Agreed: there are likely to be many Intel boards, but this only supports one of them for now. Maybe this should be "up4000", or maybe the plan is to expand this to support lots of boards by having lots of definitions in data.go? but I'd like a plan for how this works before we commit to such a broad model name.

Reply:
Will update this when we decide on the name in scope doc. Till then, I'll keep "intel" as a placeholder.